__tests__/app.test.js

```diff
@@ -238,4 +261,54 @@ describe("GET", () => {
         });
       });
   });
+
+  test("/api/articles/:article_id/comments 200: returns the comments of the specific article as requested by the client.", () => {
+    return request(app)
+      .get("/api/articles/3/comments")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body).toEqual({
+          commentsOfThisArticle: [
+            {
+              comment_id: 11,
+              body: "Ambidextrous marsupial",
+              article_id: 3,
+              author: "icellusedkars",
+              votes: 0,
+              created_at: "2020-09-19T23:10:00.000Z",
+            },
+            {
+              comment_id: 10,
+              body: "git push origin master",
+              article_id: 3,
+              author: "icellusedkars",
+              votes: 0,
+              created_at: "2020-06-20T07:24:00.000Z",
+            },
+          ],
+        });
+      });
+  });
```

Comment:
This test is valid but you ar free to just test the length of the array and that the properties inside the objects are all there